caffe2/python/benchmark_generator.py

```diff
@@ -85,7 +85,7 @@ def make_blob_on_context(blob_name, blob_data, context):
 
     # Handle manual rewrite
     if args.context.upper() == "OPENGL":
-        old_ops = [op for op in predict_net.op]
+        old_ops = list(op for op in predict_net.op)
```

Comment:
Just curious, why this change is necesary?

Reply:
Not needed, by when I ran `make mypy`, I got a list of issues in files I have changed and this was another occurrence. Maybe my local version was too new or too old, or at least different than the ones PyTorch pipelines use?